keyboards/noxary/268_2/n268_2.c

```diff
@@ -0,0 +1,53 @@
+/* Copyright 2018 Rozakiin
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "n268_2.h"
```

Comment:
Shouldn't this be `#include QMK_KEYBOARD_H`?

Reply:
We've only been changing that in `keymap.c` so far. We should change the `new_project.sh` script to reflect this if we want to do this too.

Reply:
Oops, you're right, I misread the filename.